README.md

```diff
@@ -23,6 +23,10 @@ A lightweight page object implementation with a focus on simplicity and extensib
     - [Lightweight (no jQuery)](#lightweight-no-jquery)
 - [Why `fractal`?](#why-fractal)
 
+## Example
+
+Using plain [ember-qunit](https://github.com/emberjs/ember-qunit):
```

Comment:
This isn't actually using `ember-qunit` or anything Ember, just `qunit`. So the intended distinction here is between non-Ember (using vanilla `qunit` as an example) and using Ember (`@ember/test-helpers` and `qunit-dom`).

I'm happy to take a PR like this to clarify things if they seem unclear, but although this library has Ember's testing framework in mind, it is framework-agnostic and I want to make sure that message isn't lost.

Reply:
Cool. I can reword. I think the heading is a good idea, because the code is there without any explanation. And then there are two blocks that seem very similar so explaining what is the difference IMO makes sense.